monitor/ui/pom.xml

```diff
@@ -208,13 +208,13 @@
 									<goal>start</goal>
 								</goals>
 							</execution>
-							<execution>
-								<id>stop</id>
-								<phase>post-integration-test</phase>
-								<goals>
-									<goal>stop</goal>
-								</goals>
-							</execution>
+							<!--<execution>-->
+								<!--<id>stop</id>-->
+								<!--<phase>post-integration-test</phase>-->
+								<!--<goals>-->
+									<!--<goal>stop</goal>-->
+								<!--</goals>-->
+							<!--</execution>-->
```

Comment:
Please re-institute the "stop" execution, so no Docker images keep lingering around.